src/Compilers/Core/Portable/Operations/OperationExtensions.cs

```diff
@@ -33,23 +34,45 @@ public static IEnumerable<IOperation> Descendants(this IOperation operation)
             {
                 return SpecializedCollections.EmptyEnumerable<IOperation>();
             }
-            var list = new List<IOperation>();
-            var collector = new OperationCollector(list);
-            collector.Visit(operation);
-            list.RemoveAt(0);
-            return list;
+
+            return DescendantsAndSelf(operation).Skip(1);
```

Comment:
Perhaps add a private `Descendants(IOperation operation, bool includeSelf)` that both this method and `DescendantsAndSelf` can use.